### DISCUSSION TITLE
How to print to pdf?

### DISCUSSION THREAD

**Discussion by barbalex** [tauri-apps/tauri#2591] [Q&A]
Tauri noob here, trying to move an app from electron to tauri.

Electron has a method to directly print to pdf: https://www.electronjs.org/docs/api/web-contents#contentsprinttopdfoptions

I have found similar methods in tauri:
- appWindow has a print method
- appWindow has a printToPDFOptions method (!)

But I would like to print the webview to pdf.

How could this be done?

**Comment by madisvain** [comment]
This is somewhat old question but still relevant. Is there a way to print to PDF in Tauri?

I'm rebuilding my Invoicing for freelancers applicaton [Upcount](https://www.upcount.app/) in Tauri and this would be a really nice feature to print invoices to PDF.

**Comment by cin4ed** [comment]
Well, no fucking body answered this.
